vms/volunteer/tests/test_volunteerProfile.py

```diff
@@ -302,7 +299,7 @@ def test_corrupt_resume_uploaded(self):
         try:
             PyPDF2.PdfFileReader(open(path, 'rb'))
         except PdfReadError:
-            print('Some error while upload/download')
+            self.assertTrue(False)
```

Comment:
Please use `self.fail('message')`
message as something similar to `Uploaded resume is corrupted`